### PR TITLE
Bug/2201 not list previously roles

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by DimaDiD — APPROVED** [review]

**Review by VSBalanchuk — CHANGES_REQUESTED** [review]
There are a lot of missing or extra empty lines. 
Please format your code at least in one style. 
It will improve readability of your code a lot.

File: EPlast/EPlast.BLL/Interfaces/Region/IRegionAdministrationService.cs [hunk 1]
```diff
@@ -26,5 +26,7 @@ public interface IRegionAdministrationService
 
         Task<IEnumerable<AdminTypeDTO>> GetAllAdminTypes();
         Task RemoveAdminRolesByUserIdAsync(string userId);
+
+        Task EditStatusAdministration(int adminId, bool status = false);
```
Comment (VSBalanchuk) [inline EPlast/EPlast.BLL/Interfaces/Region/IRegionAdministrationService.cs:30]: For example here.

File: EPlast/EPlast.Tests/Controllers/RegionsControllerTests.cs [hunk 2]
```diff
@@ -792,6 +792,18 @@ public async Task Remove_ReturnsRegions()
             // Assert
             Assert.IsInstanceOf<OkResult>(result);
         }
+        [Test]
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Tests/Controllers/RegionsControllerTests.cs:795]: Here

Comment (VSBalanchuk) [inline EPlast/EPlast.Tests/Controllers/RegionsControllerTests.cs:795]: Here

File: EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs [hunk 3]
```diff
@@ -123,19 +123,41 @@ public async Task GetHeadDeputy_ReturnsRegionAdministrationDTO()
...
+            // Assert
+            Assert.IsInstanceOf<int>(result);
+            Assert.AreEqual(2, result);
+            Assert.IsNotNull(result);
+        }
+
+        [Test]
+
+
         public async Task GetAdministrationAsync_ReturnsIEnumerableRegionAdministrationDTO()
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs:161]: Here

File: EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs [hunk 4]
```diff
@@ -615,6 +637,49 @@ public void AddRegionAdministrator_EndDateNull_ReturnsCorrect()
...
+            _repoWrapper.Verify(x => x.RegionAdministration.Update(It.IsAny<RegionAdministration>()), Times.Once);
+
+
+        }
+
+
+
+
+
         private static AdminTypeDTO AdminTypeHead = new AdminTypeDTO
```
Comment (VSBalanchuk) [inline EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs:683]: Here

File: EPlast/EPlast.WebApi/Controllers/RegionsController.cs [hunk 5]
```diff
@@ -684,6 +684,13 @@ public async Task<IActionResult> Remove(int Id)
             await _regionAdministrationService.DeleteAdminByIdAsync(Id);
             return Ok();
         }
+        [HttpPut("ChangeStatusAdministration/{Id}")]
```
Comment (VSBalanchuk) [inline EPlast/EPlast.WebApi/Controllers/RegionsController.cs:687]: Here

**Review by VSBalanchuk — COMMENTED** [review]
Try out auto formatting. 
For example, you can use: 
- ReSharper feature named "Cleanup code" 
- I think even ctrl+k => ctrl+f will help you a lot

File: EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs (hunk 4 above)
Comment (Goven112) [reply EPlast/EPlast.Tests/Services/Regions/RegionAdministrationServiceTests.cs]: done

File: EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs [hunk 6]
```diff
@@ -200,7 +206,6 @@ public async Task<RegionAdministrationDTO> GetHeadDeputy(int regionId)
 
             return _mapper.Map<RegionAdministration, RegionAdministrationDTO>(headDeputy);
         }
-
```
Comment (VSBalanchuk) [inline EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs:203]: Revert this change.

File: EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs [hunk 7]
```diff
@@ -132,12 +132,20 @@ public async Task<RegionAdministrationDTO> EditRegionAdministrator(RegionAdminis
...
+        {
+            var admin = await _repoWrapper.RegionAdministration.GetFirstOrDefaultAsync(a => a.ID == adminId);
+            if (admin != null)
+            {
+                admin.Status = status;
+                _repoWrapper.RegionAdministration.Update(admin);
+                await _repoWrapper.SaveAsync();
+            }
+        }
         public async Task DeleteAdminByIdAsync(int Id)
```
Comment (VSBalanchuk) [inline EPlast/EPlast.BLL/Services/Region/RegionAdministrationService.cs:145]: Add empty line

**Review by VSBalanchuk — APPROVED** [review]

**Pull request merged**